### DISCUSSION TITLE
blog/web/react-props-depend-to-others

### DISCUSSION THREAD

**Discussion by giscus** [TawalMc/tawaldevuniverse#3] [Announcements]
# blog/web/react-props-depend-to-others

A tip to make some react props depend to others props.

https://tawaldevuniverse.vercel.app/blog/web/react-props-depend-to-others

**Comment by tawaliou** [comment]
Good post

**Reply by tawaliou** [reply]
If you have a bestter approach is share with us. Thanks.

**Reply by tawaliou** [reply]
better*
